### ISSUE TITLE
Post processing failing with errors due to missing files

### ISSUE THREAD

**Issue by simora** [mjbrowns/plexpost#4]
the log from the initial comchap execution doesn't exist but it completed with something stating no chapters found. looking over the change history in comchap.ini i found no obvious issues or changes other than a ";' near disable_heuristics but if it worked before than there is no reason it should have failed with proper syntax.

root@2a31bf07073c:/postdata/queue/working-169d1f23a567127ea794d294e1ca5c163fafb1dc# ls
comcut.log  comskip.ini  original.edl  original.mkv  transcode.log

comchap.edl doesn't exist

/usr/local/bin/comcut: line 173: /postdata/queue/working-169d1f23a567127ea794d294e1ca5c163fafb1dc/comchap.edl: No such file or directory

comchap.mkv doesn't exist.

comcut.log
Comskip 0.81.098, made using ffmpeg
Donator build
/postdata/queue/working-169d1f23a567127ea794d294e1ca5c163fafb1dc/comchap.mkv: Can not open file

The last line repeats for quite a while.

**Comment by simora** [comment]
This file seems to be cleaned up quickly but comchap.log ends with the following.

The commandline used was:
comskip -q -n --ini=/postdata/queue/working-beaf63a4f89d2bd0b88231dc21f97bf351ee50c8/comskip.ini /postdata/queue/working-beaf63a4f89d2bd0b88231dc21f97bf351ee50c8/original.mkv

Setting ini file to /postdata/queue/working-beaf63a4f89d2bd0b88231dc21f97bf351ee50c8/comskip.ini as per commandline
Using /postdata/queue/working-beaf63a4f89d2bd0b88231dc21f97bf351ee50c8/comskip.ini for initiation values.
Commercials were not found.
No commercials found: /postdata/queue/working-beaf63a4f89d2bd0b88231dc21f97bf351ee50c8/original.mkv

**Comment by mjbrowns** [comment]
Interestingly there's a thread going on right now over at the comskip side related to this.  I've not actually had this happen yet, so I'm not sure if I can reproduce this or come up with a workaround.  Bottom line is that if comskip does not find any commercials, it does not actually create the .edl file, and that is clearly breaking the rest of the processing.

Now, what to do about it.  I have a couple of options.  One, I could assume that if no commercials were found we should just detect that and move on.  However, that assumes that there really aren't commercials. If there really are commercials, then we should really actually be erroring out more cleanly so the user knows they have to tweak their comskip.ini files. 

I don't know if you noticed but I pushed a new release up a few days ago that has the initial bits of the capability to have custom comskip.ini files for each show.  

As it stands, you can now add another volume map for /config and put it somewhere; then create filters.cfg and set up mapping for your show names to ini files and then once you have the show tuned to your best, you can simply rerun the job.

**Comment by simora** [comment]
i have no experience in working with comskip. personally i would like it to flag the file as possibly inaccurately flagged and move on to transcoding. i would rather the file be transcoded and added back to my library and left in a "error/warning" state for further investigation and/or reprocessing where the new file supersedes the older.

**Comment by mjbrowns** [comment]
Oh crud.  I started looking at this today and I realized I actually will have to do quite a bit of rewriting to handle this well.  It also exposed some poor optimization on my part.  Originally, I used the same file names along the way so that after the original comchap run (which just marks the detected commercial points), when comcut would run it wouldn't have to run the comskip again - it would just use the same .edl file.  However when I fixed some other problems I changed some of the temporary file names around and broke that.

At any rate, I'm working on a solution.  Hopefully it won't take too long.
